content/en/Integrations/IntegrationBuilder/How to Guides/map-cobalt-pentest-finding-severity-to-jira-issue-priority.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Map Cobalt Pentest Finding Severity to Jira Issue Priority"
+linkTitle: "Map Cobalt Pentest Finding Severity to Jira Issue Priority"
+weight: 70
+---
+
+{{% pageinfo %}}
+
+This guide is designed to help [**Integration Builder**](/integrations/integrationbuilder/) users map the severity of pentest findings in Cobalt to the priority of new Jira issues.
+Please follow the [Create Jira tickets for Findings guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration) before proceeding with this guide.
+
+{{% /pageinfo %}}
+
+## Basic Option
+
+1. Navigate to the __[Cobalt > Jira Cloud] Push pentest finding to Jira__ recipe that you set up while following the previous guide
```

Comment:
Should we add a link to the previous guide?

Reply:
I added a link in the info header at the top of the page, but good idea, can't hurt to add another link here too.
Updated, thanks!